sdk/src/main/res/values/dimens.xml

```diff
@@ -11,4 +11,6 @@
     <dimen name="kinecosystem_menu_info_margin">2dp</dimen>
     <dimen name="kinecosystem_info_dot_radius">8dp</dimen>
     <dimen name="kinecosystem_radius_size">2dp</dimen>
+    <dimen name="kinecosystem_transfer_header_size">56dp</dimen>
```

Comment:
delete and use what I suggested, please.

Reply:
ok